src/AwsCryptographicMaterialProviders/Keyrings/AwsKms/AwsKmsArnParsing.dfy

```diff
@@ -15,9 +15,9 @@ module  AwsKmsArnParsing {
   const MAX_AWS_KMS_IDENTIFIER_LENGTH := 2048
 
   datatype AwsResource = AwsResource(
```

Comment:
Would:
```suggestion
  datatype AwsResource = 
    AwsResource(
```
Prevent the over-indenting that is occurring?

Reply:
Yes, exactly ! Would you like me to add this manual indentation as part of this PR?

Reply:
I could fix it manually everywhere as needed, where I see a lot of extra indentation.

Reply:
I... Want the least work for you and others for the best result.
Are you proposing that:
1. You manually reformat some of our dafny such that the formatter automatically formats it the correct way
2. Re-run the formatter? 

As long as that is tractable work for you, then yes.
But I request that you do that as two separate commits, 
such that I can see what I should do in the future to get the result I want.

Does that make sense?

Reply:
Ok. For now, I haven't pushed any manual commit, just upgraded the Dafny formatter.
Can you please check the new version?

Reply:
Yes.